recipes/globus-sdk/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "globus-sdk" %}
+{% set version = "1.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/globus/globus-sdk-python/archive/{{ version }}.tar.gz
+  sha256: 202dadb9e011d3846d71638e79717c192bc431ce8e7fc8b900adbfd4e4dd9407
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+  run:
+    - python
+    - requests >=2.0.0,<3.0.0
+    - six >=1.10.0,<2.0.0
+    - pyjwt[crypto] >=1.5.3,<2.0.0
+
+test:
+  imports:
+    - globus_sdk
+    - globus_sdk.auth
+    - globus_sdk.auth.client_types
+    - globus_sdk.authorizers
+    - globus_sdk.transfer
+    - globus_sdk.transfer.response
+
+about:
+  home: https://github.com/globus/globus-sdk-python
+  license: Apache 2.0
+  license_family: APACHE
+  license_file: LICENSE
+  summary: Globus SDK for Python
+  description: This SDK provides a convenient Pythonic interface to Globus APIs.
+  doc_url: http://globus-sdk-python.readthedocs.io/
+  dev_url: https://github.com/globus/globus-sdk-python
+
+extra:
+  recipe-maintainers:
+    - xylar
+    - lukaszlacinski
+    - rpwagner
```

Comment:
@lukaszlacinski and @rpwagner Are you happy to be maintainers of this recipe?

Reply:
@chrisburr I am.

Reply:
Okay, I'll do that.